internal/configs/sharding/localnet.go

```diff
@@ -31,8 +31,8 @@ const (
 	localnetV1Epoch = 1
 
 	localnetEpochBlock1      = 5
-	localnetBlocksPerEpoch   = 64
-	localnetBlocksPerEpochV2 = 64
+	localnetBlocksPerEpoch   = 8
+	localnetBlocksPerEpochV2 = 8
```

Comment:
reducing to 8 will impact the external external validator

Reply:
16 ?

Reply:
based on testing for external it had to be 64